### PR TITLE
N5Utils.saveRegion writes into an existing dataset

### PR DESCRIPTION
**Pull request by bogovicj**
* add test of this method

### CODE REVIEW

**Comment by bogovicj** [comment]
pass DatasetAttributes to saveRegion

**Review by axtimwalde — CHANGES_REQUESTED** [review]
ExecutorService is not used to parallelize as far as I can see.

**Comment by axtimwalde** [comment]
That ready to go?

File: src/main/java/org/janelia/saalfeldlab/n5/imglib2/N5Utils.java [hunk 1]
```diff
@@ -1390,6 +1397,274 @@ static void cropBlockDimensions(
 			f.get();
 	}
 
+	public static <T extends NativeType<T>> void saveRegion(
```
Comment (axtimwalde) [inline src/main/java/org/janelia/saalfeldlab/n5/imglib2/N5Utils.java:1400]: Add Javadoc.

File: src/main/java/org/janelia/saalfeldlab/n5/imglib2/N5Utils.java [hunk 2]
```diff
@@ -1390,6 +1397,274 @@ static void cropBlockDimensions(
...
+			final RandomAccessibleInterval<T> source,
+			final N5Writer n5,
+			final String dataset ) throws IOException, InterruptedException, ExecutionException
+	{
+		saveRegion( source, n5, dataset, n5.getDatasetAttributes( dataset ) );
+	}
+
+	public static <T extends NativeType<T>> void saveRegion(
```
Comment (axtimwalde) [inline src/main/java/org/janelia/saalfeldlab/n5/imglib2/N5Utils.java:1408]: Add Javadoc.

File: src/main/java/org/janelia/saalfeldlab/n5/imglib2/N5Utils.java (hunk 1 above)
Comment (bogovicj) [reply src/main/java/org/janelia/saalfeldlab/n5/imglib2/N5Utils.java:1416]: Added in 2696c84

**Comment by bogovicj** [comment]
Think this is good to go now @axtimwalde

**Comment by axtimwalde** [comment]
Thanks a lot!

**Pull request merged**